files/common/etc/update-motd.d/00-delphix

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+full_version=$(/usr/bin/get-appliance-version)
+version=$(echo "$full_version" | awk -F'-' '{print $1}')
+build=$(echo "$full_version" | sed 's/.*-\(.*\)+.*/\1/')
```

Comment:
I think we should display anything before the `+` sign as the version, so your example would become:
```
Welcome to Delphix 6.1.0.0-snapshot.20191127042627846
```
Most release builds will have an empty "build" field.

I've been using [this doc](https://docs.google.com/document/d/1K9-7TY0LdLtdT-MilECptO_6taH3edrz5pjH--LBy3s/edit#heading=h.pc1842v34c8x) as reference.

Reply:
You're right, thanks for pointing that out. I'll take the suggestion and simply print everything before the `+` sign as-is.